src/transformers/integrations/bitsandbytes.py

```diff
@@ -21,7 +21,9 @@
 logger = logging.get_logger(__name__)
 
 
-def set_module_quantized_tensor_to_device(module, tensor_name, device, value=None, fp16_statistics=None):
+def set_module_quantized_tensor_to_device(
+    module, tensor_name, device, value=None, fp16_statistics=None, quantized_stats=None
```

Comment:
The best solution would have been to only have one arg to store the quantized stats for 8-bit or 4-bit but i'm fine having both args as it is clearer.

Reply:
agreed and done